<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QEPvLoadSaveTimeDialog</class>
 <widget class="QWidget" name="QEPvLoadSaveTimeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Date Time Select</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>276</width>
       <height>290</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QCalendarWidget" name="dateCalendar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>240</width>
          <height>180</height>
         </size>
        </property>
        <property name="minimumDate">
         <date>
          <year>2005</year>
          <month>8</month>
          <day>1</day>
         </date>
        </property>
        <property name="maximumDate">
         <date>
          <year>2099</year>
          <month>12</month>
          <day>31</day>
         </date>
        </property>
        <property name="firstDayOfWeek">
         <enum>Qt::Monday</enum>
        </property>
        <property name="gridVisible">
         <bool>true</bool>
        </property>
        <property name="horizontalHeaderFormat">
         <enum>QCalendarWidget::SingleLetterDayNames</enum>
        </property>
        <property name="verticalHeaderFormat">
         <enum>QCalendarWidget::NoVerticalHeader</enum>
        </property>
        <property name="dateEditAcceptDelay">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="timeSlider">
        <property name="maximum">
         <number>239</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="minimumSize">
         <size>
          <width>168</width>
          <height>0</height>
         </size>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
